### PR TITLE
Draft features

### PR DESCRIPTION
**Pull request by Gnwin**
The branch is ready for perusal and awaiting feedback.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
**Status: changes required
Hello TEAM :wave:** 
You have done a terrific job to create a perfect layout fulfilling most of the requirements. But some minor changes are required to get approved. Let's make the following change and request another code review.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+                <input type="text" placeholder="Search" value=""/>
+                <button><i class="fa fa-search"></i></button>
+            </form>
+
+            <i class="fa fa-upload fa-lg" aria-hidden="true"></i>
+            
+            <i class="fa fa-th"></i>
+
+            <i class="fa fa-user fa-lg"></i> 
+
```
Comment (sumancrest0001) [inline index.html]: 1. There should be three images on the right side of the search button. Currently, you are using font awesome icons. [pathright screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+                            </div>
+                            
+                            <div class="views" style="display: flex; align-items: flex-end;">
+                                <span class="watch">50,016 views</span> 
+                            </div>
+                        </div>
+                        <hr>
+                        <div class="share">
+                            <a class="sh"><i class="fa fa-plus fa-lg" aria-hidden="true"></i>Add to</a>
+                            <a class="sh"><i class="fa fa-share fa-lg" aria-hidden="true"></i>Share</a>
```
Comment (sumancrest0001) [inline index.html]: 2. There should be a share button below the video content but currently, it is an icon.

**Review by sumancrest0001 — COMMENTED** [review]
3. Please make the PR descriptive and commit messages should be in the present tense.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Kindly make the above-requested changes.**
If you have any queries regarding the review, message me on slack **Suman Shrestha**
Happy coding !!!

File: index.html (hunk 1 above)
Comment (Gnwin) [reply index.html]: Three icons on the right side of the search button have been replaced with images.
Thanks. Awaiting approval.

File: index.html (hunk 2 above)
Comment (Gnwin) [reply index.html]: The share button below the video content is no more an icon. It is now a button.
Thanks. Awaiting approval.

**Pull request closed without merging**